app/models/validator.rb

```diff
@@ -67,7 +67,7 @@ def index_order(order_param)
                      'validator_score_v1s.total_score desc, RAND()'
                    end
 
-      includes(:validator_score_v1).order(sort_order)
+      order(sort_order)
```

Comment:
This method won't work unless validator_score_v1 is joined before. This potential error should be handled somehow.